src/integrationtests/java/com/aws/greengrass/integrationtests/provisioning/TestDeviceProvisioningPlugin.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.integrationtests.provisioning;
+
+import com.aws.greengrass.provisioning.DeviceIdentityInterface;
+import com.aws.greengrass.provisioning.ProvisionConfiguration;
+import com.aws.greengrass.provisioning.ProvisionContext;
+import com.aws.greengrass.provisioning.exceptions.RetryableProvisioningException;
+
+import java.util.Map;
+
+public class TestDeviceProvisioningPlugin implements DeviceIdentityInterface {
+    private static final String SERVICE_NAME = "aws.greengrass.TestProvisioningPlugin";
+
+    @Override
+    public ProvisionConfiguration updateIdentityConfiguration(ProvisionContext provisionContext) throws RetryableProvisioningException {
+        Map<String, Object> parameterMap = provisionContext.getParameterMap();
+        ProvisionConfiguration.SystemConfiguration systemConfiguration =
+                new ProvisionConfiguration.SystemConfiguration();
+        systemConfiguration.setCertificateFilePath(parameterMap.get("certificateFilePath") == null ? ""
```

Comment:
these should be consts in the deviceconfiguration class

Reply:
This is a test plugin which is independent of those constants and is dealing with its own parameters. Any change in device configuration constants should not change the values here.